containers/containers.cabal

```diff
@@ -32,7 +32,7 @@ source-repository head
     location: http://github.com/haskell/containers.git
 
 Library
-    build-depends: base >= 4.6 && < 5, array >= 0.4.0.0, deepseq >= 1.2 && < 1.5
+    build-depends: base >= 4.7 && < 5, array >= 0.4.0.0, deepseq >= 1.2 && < 1.5
```

Comment:
Could you also remove 7.6.3 from the `tested-with` section above?

Reply:
done